tensorboard/webapp/plugins/npmi/views/selected_annotations/selected_annotations_component.scss

```diff
@@ -0,0 +1,37 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+@import 'tensorboard/webapp/theme/tb_theme';
+
+.pc-container {
+  background-color: mat-color($mat-light-theme-background, card);
```

Comment:
No, neither of those work as is. I added another variable to `tensorboard/webapp/theme/_tb_theme.template.scss`, I am not sure if this is what should be done, however.

Reply:
Hm, I tried it out locally and 

```
$_background: map-get($tb-theme, background);
  background-color: mat-color($_background, card);
```
works for me.

That said, `$tb-background` is a convenient shorthand, that's reasonable since we already have `$tb-foreground`.
I've added a comment below on the tb_theme.template.scss change.